prov/hook/trace/src/hook_trace.c

```diff
@@ -150,6 +150,34 @@ trace_cq(struct hook_cq *cq, const char *func, int line,
 	}
 }
 
+static inline void
+trace_cq_err(struct hook_cq *cq, const char *func, int line,
+             int count, struct fi_cq_err_entry *entry,  uint64_t flags)
+{
+	if ((count > 0) &&
```

Comment:
This if forces the entire function to be indented.  You could move the ```if (ret > 0)``` portion of the check to the caller (or keep it here)